benchmarking/process_json_to_csv.jl

```diff
@@ -131,9 +131,6 @@ function read_dataset(dataset_list::DatasetList)::DataFrame
       join([dataset.config[c] for c in dataset_list.config_labels], ",")
 
     log_files = Glob.glob("*_summary.json", logs_directory)
-    if length(log_files) == 0
```

Comment:
I think it's still useful to print a warning for configs that have no _summary.json files - they could be a symptom of some other problem than just processing prior to completion of a run.

Reply:
Added the old message back as a warning